forms-flow-web/src/components/Modeler/DecisionTable.js

```diff
@@ -32,7 +32,12 @@ const DecisionTable = React.memo(() => {
   const [searchDmnLoading, setSearchDmnLoading] = useState(false);
   const redirectUrl = MULTITENANCY_ENABLED ? `/tenant/${tenantKey}/` : "/";
   const [search, setSearch] = useState(searchText || "");
-
+  
+  useEffect(() => {
+    if (!search?.trim()) {
+      dispatch(setBpmnSearchText(""));
```

Comment:
@fahad-aot can u check this lint error. seems like this varible is not declared

Reply:
Fixed